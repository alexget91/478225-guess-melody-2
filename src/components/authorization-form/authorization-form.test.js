import React from "react";
import renderer from "react-test-renderer";
import AuthorizationForm from "./authorization-form";

it(`Authorization form correctly renders after relaunch`, () => {
  const tree = renderer
    .create(<AuthorizationForm onFormSubmit={jest.fn()}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
