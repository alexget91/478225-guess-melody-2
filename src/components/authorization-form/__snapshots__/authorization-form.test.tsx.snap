// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Authorization form correctly renders after relaunch 1`] = `
<form
  action=""
  className="login__form js-login-form"
  onSubmit={[Function]}
>
  <p
    className="login__field"
  >
    <label
      className="login__label"
      htmlFor="name"
    >
      Логин
    </label>
    <input
      className="login__input js-email-input"
      id="name"
      name="name"
      required={true}
      type="email"
    />
  </p>
  <p
    className="login__field"
  >
    <label
      className="login__label"
      htmlFor="password"
    >
      Пароль
    </label>
    <input
      className="login__input js-password-input"
      id="password"
      name="password"
      required={true}
      type="password"
    />
    <span
      className="login__error"
    >
      Неверный пароль
    </span>
  </p>
  <button
    className="login__button button"
    type="submit"
  >
    Войти
  </button>
</form>
`;
